### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by jonasfreimuth**
Minor fixes and changes. Will get the pipeline to the state described in PR BIMSBbioinfo/pigx_docs#15 of the pigx_docs.

### CODE REVIEW

File: tests/setup_test_settings.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+locations:
```
Comment (rekado) [inline tests/setup_test_settings.yaml:1]: This file should not be included.  It should be generated from the corresponding .in file.

File: snakefile.py [hunk 2]
```diff
@@ -376,6 +374,7 @@ def render_qc_report_params(wildcards, input, output = None, threads = None, res
 KRAKEN_DIR        = os.path.join(OUTPUT_DIR, 'kraken')
 COVERAGE_DIR      = os.path.join(OUTPUT_DIR, 'coverage')
 REPORT_DIR        = os.path.join(OUTPUT_DIR, 'report')
+FASTP_DIR         = os.path.join(OUTPUT_DIR, 'fastp')
 FASTQC_DIR        = os.path.join(REPORT_DIR, 'fastqc')
```
Comment (rekado) [inline snakefile.py:378]: Is FASTQC_DIR still used anywhere?  Or is it all FASTP_DIR now?

File: snakefile.py [hunk 3]
```diff
@@ -321,7 +319,7 @@ def render_qc_report_params(wildcards, input, output = None, threads = None, res
 MUTATION_SHEET_CSV  = config["locations"]["mutation-sheet"]
 INPUT_DIR           = config["locations"]["input-dir"]
 REFERENCE_FASTA     = config["locations"]["reference-fasta"]
-AMPLICONS_BED       = config["locations"]["amplicons-bed"]
+PRIMERS_BED       = config["locations"]["primers-bed"]
```
Comment (rekado) [inline snakefile.py]: nitpick: indentation

File: etc/settings.yaml.in [hunk 4]
```diff
@@ -50,29 +51,29 @@ parameters:
...
+    #     Robust linear implemented in the `MASS` package.
+    #   rlm:
+    #     Robust linear regression as implemented in the `MASS` package, executed
+    #     via the `deconvR` package.
+    #   nnls:
+    #     Non-negative least squares (`deconvR`).
+    #   qp:
+    #     Quadratic programming (`deconvR`).
+    #   svr:
+    #     Support vector regression (`deconvR`).
```
Comment (rekado) [inline etc/settings.yaml.in]: I'm not a fan of moving the comments inside the block, but I don't feel strongly enough about this to argue :)

**Review by rekado — CHANGES_REQUESTED** [review]
The generated yaml file should be excluded.  Otherwise this looks good to me.

File: tests/setup_test_settings.yaml (hunk 1 above)
Comment (jonasfreimuth) [reply tests/setup_test_settings.yaml:1]: It is explicitly included to allow simple downloading of the repo and running a setup check with manually installed databases *without* entering of a build environment and configuring. I have clarified that [section](https://jonasfreimuth.github.io/pigx_docs/pigx-sars-cov-2.html#quick-start) in the docs a bit.

File: snakefile.py (hunk 2 above)
Comment (jonasfreimuth) [reply snakefile.py:378]: Yes, I just wanted to make sure that the fastp output can be easily found. I felt it was confusing to put the output of one tool into the dir of another.

File: snakefile.py (hunk 3 above)
Comment (jonasfreimuth) [reply snakefile.py]: Fixed

File: etc/settings.yaml.in (hunk 4 above)
Comment (jonasfreimuth) [reply etc/settings.yaml.in]: Hmm I removed all the comments, as everything should be explained in the docs anyways...

File: tests/setup_test_settings.yaml (hunk 1 above)
Comment (rekado) [reply tests/setup_test_settings.yaml:1]: Okay, got it.

**Pull request merged**